CHANGELOG.md

```diff
@@ -68,6 +68,10 @@
   * `cortex_ingester_tsdb_wal_truncations_failed_total`
   * `cortex_ingester_tsdb_wal_truncations_total`
   * `cortex_ingester_tsdb_wal_writes_failed_total`
+  * `cortex_ingester_checkpoint_deletions_failed_total`
+  * `cortex_ingester_checkpoint_deletions_total`
+  * `cortex_ingester_checkpoint_creations_failed_total`
+  * `cortex_ingester_checkpoint_creations_total`
```

Comment:
Should we use `_tsdb_` in the name of these metrics?

Reply:
Definitely yes! Thanks for spotting the mistake 🤦